pino.js

```diff
@@ -203,7 +209,12 @@ Object.defineProperty(Pino.prototype, 'levelVal', {
 })
 
 Pino.prototype._setLevel = function _setLevel (level) {
-  if (typeof level === 'number') { level = this.levels.labels[level] }
+  if (typeof level === 'number') {
+    if (!isFinite(level)) {
+      throw new Error('unknown level ' + level)
```

Comment:
as a matter of style, we avoid `new` if we possibly can - `Error` can be called without `new` to the same effect